README.md

```diff
@@ -1,45 +1,27 @@
+| [Remote Controller](http://github.com/material-foundation/material-remixer-remote-web) | [![Build Status](https://travis-ci.org/material-foundation/material-remixer-remote-web.svg?branch=develop)](https://travis-ci.org/material-foundation/material-remixer-remote-web) |  | [![npm version](https://badge.fury.io/js/material-remixer-remote-web.svg)](https://badge.fury.io/js/material-remixer-remote-web) |
 
-- **Booleans**
-	- Example: Use a boolean variable to hide or show a view during development.
+## Variables
 
-- **Strings**
-	- Example: Use a string variable to easily try different options for a header.
+A Remixer variable has a unique key associated with it, and can be a Boolean, Color, Number (represented as floats) or String.
 
-Remixer also supports setting a list of values a variable can be set to. For example, a color variable can be configured so that its value can only be changed to red, blue or yellow.
+Remixer lets you define a range of values (for variables of type Number), or limit values to members of a list you provide. This lets you ensure that values always _make sense_.
 
 ## Web Remote Controller
 
-If enabled, the variables defined in your app will be exposed on a remote controller website. Share the link to this site with your teammates from within the overlay or by emailing them. The remote controller allows updating the defined variables, and changes will be propagated in real-time across associated apps.
-
-## Roadmap and State of development
-
-Development of Remixer is ongoing across the supported platforms. Visit our [State of Development](https://github.com/material-foundation/material-remixer/wiki/State-of-Development) wiki which details the status of development for each platform. In addition, visit our current [Roadmap](https://github.com/material-foundation/material-remixer/wiki/Roadmap) wiki for a list of features planned for future releases.
-
-## Repositories
-
-Platform specific libraries and tools can be found in the following GitHub repos:
+A separate project, the [Remote Controller](http://github.com/material-foundation/material-remixer-remote-web), is provided for those times when by interacting with Remixer you may make it hard to see the UI you're adjusting. The Remote Controller lets you adjust variable values in a separate device so you can properly appreciate all changes.
 
-- [iOS](https://github.com/material-foundation/material-remixer-ios) - Remixer for iOS.
-- [Android](https://github.com/material-foundation/material-remixer-android) - Remixer for Android.
-- [JavaScript](https://github.com/material-foundation/material-remixer-js) - Remixer for JavaScript.
-- [Web Remote](https://github.com/material-foundation/material-remixer-remote-web) - Remixer web remote controller for all platforms.
+Since there is a bit of platform specific setup in each case, look at the documentation in your platform's Remixer Repo  and the Remote Controller's for more information.
```

Comment:
"For details on how to set up this mode in your app, please visit one of the platform specific repos."